src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -345,712 +356,129 @@ private void storeMergedFeed() throws IOException, CheckedAWSException {
      */
     private int constructMergedTable(Table table, List<FeedToMerge> feedsToMerge,
         ZipOutputStream out) throws IOException {
```

Comment:
I think it'd be helpful to not have this method throw either of these exceptions so that finer-grained error handling can occur.

Reply:
Handled in https://github.com/ibi-group/datatools-server/pull/376/commits/8179f873f950f83394f7c628131f9b2e73fc72e1.